### PR TITLE
Add a method to populate countries for FCO.

### PR DESCRIPTION
**Pull request by alixedi**
Small but gnarly. I have left comments for my 6m-later-self.

### CODE REVIEW

File: caseworker/advice/views.py [hunk 1]
```diff
@@ -187,8 +187,32 @@ def get_form(self):
         else:
             raise ValueError("Invalid advice type encountered")
 
+    def advised_countries(self):
+        """Returns a dict of countries for which advice has given by the current user."""
+        dest_types = ("end_user", "ultimate_end_user", "consignee", "third_party")
```
Comment (saruniitr) [inline caseworker/advice/views.py:192]: please rearrange in logical order `consignee, end_user, ultimate_end_user, third_party`.
Not sure about the last two which comes first.

File: caseworker/advice/views.py [hunk 2]
```diff
@@ -187,8 +187,32 @@ def get_form(self):
...
+            if dest["id"] in advised_on
+        ]
+
     def form_valid(self, form):
         data = form.cleaned_data
+        # When an FCO officer edits the advice, we don't allow for changing the countries
+        # & therefore, we render the normal forms and not the FCO ones.
+        # This means that here data here doesn't include the list of countries for which
+        # the advice should be applied and so we pop that in using a method.
+        if self.caseworker["team"]["name"] == "FCO":
```
Comment (saruniitr) [inline caseworker/advice/views.py:214]: I think team id is better here

File: caseworker/advice/views.py [hunk 3]
```diff
@@ -187,8 +187,32 @@ def get_form(self):
         else:
             raise ValueError("Invalid advice type encountered")
 
+    def advised_countries(self):
+        """Returns a dict of countries for which advice has given by the current user."""
```
Comment (wkeeling) [inline caseworker/advice/views.py:191]: Looks like the method returns a `list` rather than a `dict`?

File: caseworker/advice/views.py (hunk 1 above)
Comment (wkeeling) [reply caseworker/advice/views.py:192]: Maybe it's worth moving this tuple of destination types to `constants.py` - I think `CaseContextMixin.unadvised_countries()` also uses it.

File: caseworker/advice/views.py (hunk 2 above)
Comment (alixedi) [reply caseworker/advice/views.py:214]: Is `team-id` stable across envs?

Comment (saruniitr) [reply caseworker/advice/views.py:214]: not entirely but better than name. We need to update the name soon for this.

**Review by saruniitr — APPROVED** [review]

**Pull request merged**